src/plugins/aragon/index.ts

```diff
@@ -197,7 +197,7 @@ async function scheduleAction(
   const config = result.registryEntry.queue.config;
 
   // Building the nonce for the next tx
-  const nonce = await web3.getTransactionCount(account, 'pending');
+  const nonce = await call(web3, abi, [result.registryEntry.queue.address, 'nonce', ''])
```

Comment:
You'll probably need to update the ABI code here to include the nonce method, this part https://github.com/aragon/govern/blob/e98ce3613b1fb93f5ce103ddf5f4dea3a1846077/packages/govern-console/src/lib/abi/GovernQueue.json#L1281